### PR TITLE
Fixes so that the api will build

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by ggam** [comment]
Thanks @Cousjava for looking at these. I made the changes without compiling. One question though, have you verified the OSGi stuff still works after totally commenting out the spec plugin? We're struggling with it on JSON-P: https://github.com/eclipse-ee4j/jsonp/pull/109

The solution in that case is to remove just the `check-module` goal.

**Comment by Cousjava** [comment]
@ggam I've tested it as a replacement for javax.ejb-api in Payara (which is an OSGi environment) and it worked

**Comment by ggam** [comment]
Thanks!

File: pom.xml [hunk 1]
```diff
@@ -98,11 +98,12 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.7.0</version>
             </plugin>
-            <plugin>
+             <plugin>
                 <groupId>org.glassfish.build</groupId>
                 <artifactId>spec-version-maven-plugin</artifactId>
-                <version>1.2</version>
+                <version>1.5</version>
```
Comment (Tomas-Kraus) [inline pom.xml]: This is already done.

File: pom.xml [hunk 2]
```diff
@@ -123,7 +124,7 @@
             <plugin>
                 <groupId>org.glassfish.build</groupId>
                 <artifactId>glassfishbuild-maven-plugin</artifactId>
-                <version>3.2.24</version>
+                <version>3.2.26</version>
```
Comment (Tomas-Kraus) [inline pom.xml]: This is already done

File: pom.xml [hunk 3]
```diff
@@ -308,20 +309,6 @@
                     </reporting>
                 </configuration>
             </plugin>
-            <plugin>
```
Comment (Tomas-Kraus) [inline pom.xml]: GPG plugin is configured in paren pom in oss-release profile. Child project must specify GPG plugin version only. I already did this in my PR.

File: pom.xml [hunk 4]
```diff
@@ -337,4 +324,29 @@
             <version>1.3</version>
         </dependency>
     </dependencies>
+
+   <profiles>
+        <profile>
+            <id>release</id>
```
Comment (Tomas-Kraus) [inline pom.xml]: Parent oss-release profile already does GPG plugin setup. This is not needed.

**Pull request closed without merging**